test/mocks/token-util-mock.js

```diff
@@ -34,15 +34,15 @@ function decodeAndValidate(tokenString) {
   if (tokenString === "invalid_token") {
 	deferred.resolve();
   } else if (tokenString === "bad_scope") {
-	deferred.resolve({scope: "bad_scope"});
+	deferred.resolve({scope: "bad_scope", audience: "myCliendId"});
```

Comment:
I think the audience field inside the token should be 'aud' instead of 'audience'

Reply:
changed.